runeq/v2sdk/stream.py

```diff
@@ -0,0 +1,60 @@
+"""
+V2 SDK functionality to support Stream operations.
+
+"""
+from runeq import errors
+from . import client
+
+
+def get_stream(stream_id: str, **params):
+    """
+    Fetch stream data from the Stream API.
+
+    Parameters
+    ----------
+    stream_id : str
+        Unique string identifier for a stream.
+
+    **params
+        limit: Limits the number of points returned for the stream.
+        format: 'csv' or 'json'. Defaults to csv.
+
+    Returns
+    -------
+    Generator over stream data in the specified format, defaulting to csv.
+
+    """
+    if not client._stream_client:
+        raise errors.InitializationError("Failed to initialize stream client.")
```

Comment:
I feel like the following could be a more useful message, especially for the target audience of these helper functions (non-developers), something like:
```suggestion
        raise errors.InitializationError("runeq must be initialized by calling `initialize` before this function can be used")
```

Even for a developer, this isn't a failure to initialized, it's just that the client isn't initialized, so at this point it's misleading to say it "failed".